lua/git-worktree/init.lua

```diff
@@ -6,15 +6,79 @@ local Status = require("git-worktree.status")
 
 local status = Status:new()
 local M = {}
-local git_worktree_root = vim.loop.cwd()
+local git_worktree_root = nil
 local on_change_callbacks = {}
 
+M._find_git_root_job = function()
+    local cwd = vim.loop.cwd()
```

Comment:
do we cache the results of this?  Trying to find it in the codes!

Reply:
the function sets the local file variable git_worktree_root that the other files use